spec/unit/hanami/utils/files_spec.rb

```diff
@@ -246,6 +246,19 @@ class Unshift
       expect(path).to have_content(expected)
     end
 
+    # https://github.com/hanami/utils/issues/348
+    it "adds a line at the top of a file that doesn't end with a newline" do
```

Comment:
This should be "add a line at the *bottom* of a file..."

Reply:
I've added a spec for `.unshift` as well, which adds a line at the _top_ of the file. The spec for `.append` is at line 296. 😉